centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -154,6 +169,7 @@ public void onClosing(WebSocket webSocket, int code, String reason) {
             @Override
             public void onClosed(WebSocket webSocket, int code, String reason) {
                 super.onClosed(webSocket, code, reason);
+                //Clear serverSide sub
```

Comment:
unnecessary comment left here

Reply:
sorry,  and i remove unnecessary comment.